iati/home/templatetags/iati_tags.py

```diff
@@ -54,6 +58,44 @@ def translation_links(context, calling_page):
     }
 
 
+@register.filter
+def haspassed(value):
+    """Takes a date and tells you if it's in the past"""
+    now = timezone.now()
+    return value < now
+
+
+@register.filter
+def twopartdate(date_start, date_end):
+    """Takes two datetimes and determines whether to display start and end times, or start and end dates.
+
+    If an end date exists, we can compare the two dates.
+    If the two dates are the same, localize the date for the first part and stringify the time range for the second.
+    If the two dates are not the same, part 2 becomes the second date.
+
+    If no end date exists, part 2 is just the start time.
+    """
+    part1 = _date(date_start, "DATE_FORMAT")
+    enDash = u'\u2013'
```

Comment:
My local linters are flagging this as an invalid variable name as it's in camel case instead of snake case.